build.gradle

```diff
@@ -1,25 +1,22 @@
 buildscript {
     repositories {
-        mavenCentral()
-        maven {
-            url "http://download.crashlytics.com/maven"
-        }
+        jcenter()
+        maven { url 'https://maven.fabric.io/public' }
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:1.3.0"
-        classpath "com.crashlytics.tools.gradle:crashlytics-gradle:1.14.7"
+        classpath "com.android.tools.build:gradle:2.2.3"
         classpath "com.jakewharton.sdkmanager:gradle-plugin:0.12.0"
```

Comment:
You might want to remove the sdkmanager as the gradle tools take all the necessary prerequisites.

Source: 
https://github.com/JakeWharton/sdk-manager-plugin